### DISCUSSION TITLE
Support for PS3 HD versions

### DISCUSSION THREAD

**Discussion by asw52692** [open-goal/jak-project#3216] [Q&A] [answered]
Hi there. I do not have the original PS2 versions of the original Jak and Daxter trilogy so I have no way to make ISOs for use with OpenGOAL. I do have the PS3 HD versions of the games though. Are there any plans to add support for the PS3 versions of Jax and Daxter as a source to extract the files from in order to have the necessary files to play the games in OpenGOAL?

**Comment by Zedb0T** [comment, marked as answer]
Nope, only the only planned versions are retail ps2 copies of the game.
